1.data-exploration/scripts/3.gene_dependency_subsetting.py

```diff
@@ -1,44 +1,44 @@
 #!/usr/bin/env python
 # coding: utf-8
 
-# In[1]:
+# In[111]:
```

Comment:
Same comment as above script, recommend confirming the script is ran sequentially.